wasmcloud-test-util/Cargo.toml

```diff
@@ -10,10 +10,10 @@ repository = "https://github.com/wasmcloud/wasmcloud-test"
 readme = "README.md"
```

Comment:
we'll need to bump the version to 0.10.0 before releasing

Reply:
Sure going to put it in here just to make things faster.